gir-files/GstPbutils-1.0.gir

```diff
@@ -1684,8 +1684,11 @@ information like name, description, format and preset.</doc>
             <doc xml:space="preserve">The name of the target</doc>
             <type name="utf8" c:type="const gchar*"/>
           </parameter>
-          <parameter name="profilename" transfer-ownership="none">
-            <doc xml:space="preserve">(allow-none): The name of the profile, if %NULL
+          <parameter name="profilename"
+                     transfer-ownership="none"
+                     nullable="1"
+                     allow-none="1">
+            <doc xml:space="preserve">The name of the profile, if %NULL
```

Comment:
Please also fix on the C side

Reply:
I didn't get what is wrong with this function (encoding_profile_find).

Reply:
I'll check

Reply:
What do you mean with it being wrong?

Also please don't forget to add annotation fixes to GStreamer as needed :)